### ISSUE TITLE
qml support ?

### ISSUE THREAD

**Issue by mediabuff** [copperspice/copperspice#140]
Is there going to be full fledged qml support ? I see the qml sections in 'kitchensink' are commented out

**Comment by bgeller** [comment]
Yes, there are several developers who are working on a new CsDeclarative library. The aim of QML was to solve a specific need, however the internal design is a bit limited. Our API will be have more flexibility and a good C++ interface.  

If you or anyone else would like to be more involved, feel free to contact the team. 

Barbara

**Comment by mediabuff** [comment]
Would be glad to participate and help.
